testdata/regresstest/config

```diff
@@ -435,3 +435,15 @@ check_compiler_output ../../grammar2 testtermarg.gap testme termarg grep "6), \"
 GRAMMAR=../../grammar
 GAPC="../../../gapc --float-accuracy 2"
 check_new_old_eq elm.gap unused pareto "1+2*3+4*5" floatacc
+
+GAPC="../../../gapc --checkpoint"
+RUN_CPP_FLAGS="--checkpointInterval 0:0:0:5 --checkpointInput ."
+check_checkpoint_eq adpf.gap unused mfe "UAACCCGGUCACAUGGGAUGAGGAGAUAACAUAAUCUCCCUCCCACAAGCAGUAACUAUAAAAAUAACCCCACUCUCUACAAGGCUCGGGGCGCCCGAAAAAACGGGCAUACAGGUUGACCGACAACGAUAUAAAUCGGAAUCAAAAACUAUGUGUGGAAUUGUUGGCGCGAUCGCGCAACGUGAUGUAGCAGAAAUCCUUCUUGAAGGUUUACGUCGUCUGGAAUACCGCGGAUAUGACUCUGCCGGUCUGGCCGUUGUUGAUGCAGAAGGUCAUAUGACCCGCCUGCGUCGCCUCGGUAAAGUCCAGAUGCUGGCACAGGCAGCGGAAGAACAUCCUCUGCAUGGCGGCACUGGUAUUGCUCACACUCGCUGGGCGACCCACGGUGAACCUUCAGAAGUGAAUGCGCAUCCGCAUGUUUCUGAACACAUUGUGGUGGUGCAUAACGGCAUCAUCGAAAACCAUGAACCGCUGCGUGAAGAGCUAAAAGCGCGUGGCUAUACCUUCGUUUCUGAAACCGACACCGAAGUGAUUGCCCAUCUGGUGAACUGGGAGCUGAAACAAGGCGGGACUCUGCGUGAGGCCGUUCUGCGUGCUAUCCCGCAGCUGCGUGGUGCGUACGGUACAGUGAUCAUGGACUCCCGUCACCCGGAUACCCUGCUGGCGGCACGUUCUGGUAGUCCGCUGGUGAUUGGCCUGGGGAUGGGCGAAAACUUUAUCGCUUCUGACCAGCUGGCGCUGUUGCCGGUGACCCGUCGCUUUAUCUUCCUUGAAGAGGGCGAUAUUGCGGAAAUCACUCGCCGUUCGGUAAACAUCUUCGAUAAAACUGGCGCGGAAGUAAAACGUCAGGAUAUCGAAUCCAAUCUGCAAUAUGACGCGGGCGAUAAAGGCAUUUACCGUCACUACAUGCAGAAAGAGAUCUACGAACAGCCGAACGCGAUCAAAAACACCCUUACCGGACGCAUCAGCCACGGUCAGGUUGAUUUAAGCGAGCUGGGACCGAACGCCGACGAACUGCUGUCGAAGGUUGAGCAUAUUCAGAUCCUCGCCUGUGGUACUUCUUAUAACUCCGGUAUGGUUUCCCGCUACUGGUUUGAAUCGCUAGCAGGUAUUCCGUGCGACGUCGAAAUCGCCUCUGAAUUCCGCUAUCGCAAAUCUGCCGUGCGUCGUAACAGCCUGAUGAUCACCUUGUCACAGUCUGGCGAAACCGCGGAUACCCUGGCUGGCCUGCGUCUGUCGAAAGAGCUGGGUUACCUUGGUUCACUGGCAAUCUGUAACGUUCCGGGUUCUUCUCUGGUGCGCGAAUCCGAUCUGGCGCUAAUGACCAACGCGGGUACAGAAAUCGGCGUGGCAUCCACUAAAGCAUUCACCACUCAGUUAACUGUGCUGUUGAUGCUGGUGGCGAAGCUGUCUCGCCUGAAAGGUCUGGAUGCCUCCAUUGAACAUGACAUCGUGCAUGGUCUGCAGGCGCUGCCGAGCCGUAUUGA" checkpoint1 5
```

Comment:
I know its delicate, but would it be possible to use shorter inputs to reduce overall runtime of these three tests. They consume ~1minute on my laptop - and I don't want the test suite to burn too much time. On the other hand, we cannot perfectly predict how long the cloud machines need for this. But if we reduce the interval to 0:0:0:1 we might be on the same side?!

Reply:
No problem. I shortened the test inputs and updated the Truth files on the `gapc-test-suite` branch.